app/src/components/features/mocksV2/MockEditorIndex/Editor/index.tsx

```diff
@@ -405,6 +410,8 @@ const MockEditor: React.FC<Props> = ({
             handleChange={setBody}
             language={getEditorLanguage(fileType)}
             isReadOnly={isEditorReadOnly}
+            isFullScreen={fullScreen}
+            onFullScreenChange={handleFullScreenChange}
```

Comment:
If we have to pass both the state and the setter function then it should be inside the `Editor` component.